src/window.rs

```diff
@@ -1013,6 +1013,15 @@ impl Window {
         self.window.drag_window()
     }
 
+    /// Resizes the window with the left mouse button until the button is released.
+    ///
+    /// There's no guarantee that this will work unless the left mouse button was pressed
+    /// immediately before this function is called.
```

Comment:
Should add `# Platform-specific` section describing the support and implications.